docs/3x/PT_WeightOnlyQuant.md

```diff
@@ -0,0 +1,154 @@
+
+PyTorch Weight Only Quantization
+===============
+- [Introduction](#introduction)
+- [Usage](#usage)
+  - [Get Started](#get-started)
+    - [RTN](#rtn)
+    - [GPTQ](#gptq)
+    - [AutoRound](#autoround)
+    - [AWQ](#awq)
+    - [TEQ](#teq)
+    - [HQQ](#hqq)
+  - [Specify Quantization Rules](#specify-quantization-rules)
+  - [Saving and Loading](#saving-and-loading)
+- [Examples](#examples)
+
+
+## Introduction
+
+The INC 3x New API provides support for quantizing PyTorch models using WeightOnlyQuant.
```

Comment:
no need to mention "3x New API" since whole doc is for 3.x API.  Simply introducing functionality is enough  @Kaihui-intel

Reply:
thanks, updated https://github.com/intel/neural-compressor/pull/1824/commits/c8bdcca0b82f5625304ae76260d5922d2b0e46c0